rllib/core/torch/torch_rl_module.py

```diff
@@ -0,0 +1,18 @@
+import torch
+
+from ray.rllib.core.rl_module import RLModule
+
+
+class TorchRLModule(torch.nn.Module, RLModule):
+    def __init__(self, config):
+        super().__init__()
+        self.config = config
+
+    def forward(self, batch, **kwargs):
```

Comment:
Is there a reason we don't follow the `forward_train`, `forward_inference` API listed above. IIRC, `forward` was supposed to default to inference mode but maybe @kouroshHakha can correct me.

Reply:
For the purpose of TorchRLTrainer training part you need to have a forward() method that can be aliases to the forward_train() method. It's a requirement by DDP apparently.